examples/website/geospatial/app.js

```diff
@@ -0,0 +1,135 @@
+import React, {PureComponent} from 'react';
+import {render} from 'react-dom';
+import {StaticMap} from 'react-map-gl';
+import {get} from 'lodash';
```

Comment:
We have to avoid using extra dependencies. Our goal is to make the example as simple as possible.